davinci/src/main/kotlin/com/pingidentity/davinci/collector/SingleSelectCollector.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2025 Ping Identity. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.pingidentity.davinci.collector
+
+import kotlinx.serialization.json.JsonObject
+
+/**
+ * Class representing DROPDOWN, RADIO type with SINGLE_SELECT inputType.
+ *
+ * This class inherits from the FieldCollector class and implements the Collector interface.
```

Comment:
_This class extends the [FieldCollector] class..._

Reply:
Updated.